spec/automation/unit/method_validation/amazon_check_pre_retirement_spec.rb

```diff
@@ -37,4 +37,12 @@
     expect(ws.root['ae_result']).to eq('ok')
     expect(ws.root['vm'].power_state).to eq('off')
   end
+
```

Comment:
@billfitzgerald0120 
Can we change 
@ems  = FactoryGirl.create(:ems_vmware, :zone => @zone)

to 
@ems  = FactoryGirl.create(:ems_amazon, :zone => @zone)